app/routes/armoury/mission-board.tsx

```diff
@@ -23,25 +54,203 @@ export let loader = async ({ request }: LoaderArgs) => {
 
 export default function Missions() {
 	let { missions } = useLoaderData<typeof loader>()
+
 	return (
 		<>
 			<h1 className="sr-only">Missions</h1>
-			<div className="mx-auto my-6 grid w-2/3 grid-cols-4 gap-4">
-				{missions.map((mission) => (
-					<div key={mission.id} className="h-64 w-64 rounded bg-white shadow">
-						<div>Map: {mission.map}</div>
-						<div>Difficulty: {mission.challenge}</div>
-						<div>Circumstance: {mission.circumstance}</div>
-						<div>Credits: {mission.credits}</div>
-						<div>XP: {mission.xp}</div>
-						<div>Giver: {mission.missionGiver}</div>
-						<div>Start: {mission.start}</div>
-						<div>End: {mission.expiry}</div>
-						<div>
-							Extra rewards: {JSON.stringify(mission.extraRewards ?? {})}
+			<div className="w-full h-full flex justify-center p-12 xl:px-48 overflow-y-scroll">
+				<div className="w-full flex flex-wrap justify-evenly gap-10">
+					{missions.map((mission) => (
+						<div
+							key={mission.id}
+							className="relative w-96 h-56 rounded drop-shadow-lg text-green-100 font-montserrat"
+						>
+							<img
+								src={img_url(
+									`content/ui/textures/missions/${mission.map}_medium`
+								)}
+								alt=""
+								className="absolute w-full h-full top-0 left-0 object-cover rounded"
+							/>
+
+							<div className="h-full hover:opacity-20 transition-opacity">
+								<div
+									className="relative w-full h-full pl-9 pb-2"
+									style={{
+										background: `linear-gradient(180deg,
+											hsl(0 0% 0% / 1) 0%,
+											hsl(0 0% 0% / 0.35) 80%,
+											hsl(0 0% 0% / 0) 100%)`,
+									}}
+								>
+									<div className="h-8 flex flex-row justify-between items-center text-sm">
+										<div className="uppercase">
+											{loc(
+												`loc_mission_type_${
+													missionInfo[mission.map as keyof typeof missionInfo]
+														.mission_type
+												}_name`
+											)}
```

Comment:
localization should be done in the loader to prevent the whole json file needing to download on the client